zkevm-circuits/src/pi_circuit.rs

```diff
@@ -698,10 +999,11 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize> Circuit<F>
                             rlc(tx.value.to_le_bytes(), self.randomness),
                         ),
                         (TxFieldTag::CallDataLength, F::from(tx.call_data_len)),
-                        (
-                            TxFieldTag::TxSignHash,
```

Comment:
Why is `TxSignHash` removed?

Reply:
https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/main/zkevm-circuits/src/witness/tx.rs#L41-L118